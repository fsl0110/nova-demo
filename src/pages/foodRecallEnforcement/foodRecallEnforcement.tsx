import React from "react";
import { ReportsConnected } from "./reports/reports";
import { InitiatorsConnected } from "./initiators/initiators";

document.title = "Demo App: Food recall enforcement reports";

export class FoodRecallEnforcement extends React.PureComponent {
  render() {
    return (
      <div className="page-wrapper" data-test="FoodRecallEnforcement">
        <ReportsConnected location="reports" term="" />
        <InitiatorsConnected location="initiators" />
      </div>
    );
  }
}

export default FoodRecallEnforcement;
