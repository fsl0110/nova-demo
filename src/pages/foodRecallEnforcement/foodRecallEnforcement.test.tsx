import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { FoodRecallEnforcement } from "./foodRecallEnforcement";

it("renders toMatchSnapshot", () => {
  expect(toJson(shallow(<FoodRecallEnforcement />))).toMatchSnapshot();
});
