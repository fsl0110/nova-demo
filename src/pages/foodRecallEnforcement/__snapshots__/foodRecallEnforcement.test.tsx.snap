// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders toMatchSnapshot 1`] = `
<div
  className="page-wrapper"
>
  <Connect(Reports)
    location="reports"
    term=""
  />
  <Connect(Initiators)
    location="initiators"
  />
</div>
`;
